vars/odfRunSetupScript.groovy

```diff
@@ -0,0 +1,34 @@
+def call(){
```

Comment:
@aditidukle Since these are all using shell commands you can create shell script under dir /scripts/odf and directly call them from Jenkins file No need to add methods in vars file

Reply:
@aditidukle this comment is not addressed